examples/product-sample-supabase-kt/app/src/main/java/com/example/manageproducts/data/repository/impl/ImageRepositoryImpl.kt

```diff
@@ -0,0 +1,24 @@
+package com.example.manageproducts.data.repository.impl
+
+import com.example.manageproducts.BuildConfig
+import com.example.manageproducts.data.repository.ImageRepository
+import io.github.jan.supabase.storage.Storage
+import javax.inject.Inject
+
+class ImageRepositoryImpl @Inject constructor(
+    private val storage: Storage
+) : ImageRepository {
+    override suspend fun uploadImage(fileName: String, image: ByteArray): String {
+        val result =
+            storage["Product%20Image"].upload(path = "$fileName.png", data = image, upsert = false)
+        return result
+    }
+
+    override suspend fun getImage() {
+        TODO("Not yet implemented")
+    }
```

Comment:
There seems to be a missing implementation

Reply:
I removed the whole class as it is not needed anymore